doc/Administration-guide/System-installation/Single-node-installation/index.md

```diff
@@ -44,11 +44,11 @@ Kaa requires the following third party components to be installed and configured
 * [Oracle JDK 8](http://www.oracle.com/technetwork/java/javase/downloads/index.html). Kaa has been tested on JDK 8.
 * [PostgreSQL 9.4](http://www.postgresql.org/download/)
 * [MariaDB 5.5](https://mariadb.com/)
-* [Zookeeper 3.4.5](http://zookeeper.apache.org/doc/r3.4.5/). Kaa requires ZooKeeper for coordination of server components.
+* [Zookeeper 3.4.5 or later](http://zookeeper.apache.org/doc/r3.4.5/). Kaa requires ZooKeeper for coordination of server components.
 
 Kaa has been tested on the latest production release of MariaDB and PostgreSQL.
 
-Kaa also requires [MongoDB 2.6.9](http://www.mongodb.org/downloads) or [Cassandra 3.5](http://cassandra.apache.org/download/) as a NoSQL database. The installation steps for third-party components are provided in the following section.
+Kaa also requires [MongoDB 2.6.9](http://www.mongodb.org/downloads) or [Cassandra 3.5](http://cassandra.apache.org/download/) (note: for RPM based OS (Centos) 3.0.7 is the latest available version in DataStax repo) as a NoSQL database. The installation steps for third-party components are provided in the following section.
```

Comment:
In [this installation guide](http://docs.kaaproject.org/display/KAA/Installation+guide) Cassandra 3.5 is used for Centos.